src/metabase/api/database.clj

```diff
@@ -976,6 +976,7 @@
   {id ms/PositiveInt}
   ;; just wrap this in a future so it happens async
   (let [db (api/write-check (t2/select-one Database :id id))]
+    (events/publish-event! :event/database-manual-sync db)
```

Comment:
I see publish-event! as the "entrypoint" we can use for the audit system (and maybe whatever other system cares about these events). So I always called publish-event! from where the event-in-question occured.